include/hc_printf.hpp

```diff
@@ -23,23 +23,31 @@ union PrintfPacketData {
   float           f;
   void*           ptr;
   const void*     cptr;
-  std::atomic_int ai;
+
+  // Used by headers for atomic offset insts
+  std::atomic_uint ai[2];
```

Comment:
@aaronenyeshi for other types they are relatively easy to understand, but perhaps it's better to document what these fields are for. especially what `ai[0]` `ai[1]` `uia[0]` `uia[1]` means.